crypto/src/lib.rs

```diff
@@ -61,18 +62,9 @@ pub const BLS_NORMAL: &str = "bls_normal";
 /// bls small
 pub const BLS_SMALL: &str = "bls_small";
 
-/// Error indicating algorithm could not be found
-#[derive(Debug, Clone, Copy, Display, IntoSchema)]
-#[display(fmt = "Algorithm not supported")]
-pub struct NoSuchAlgorithm;
-
-#[allow(clippy::std_instead_of_alloc)]
-#[cfg(feature = "std")]
-impl std::error::Error for NoSuchAlgorithm {}
-
 ffi::ffi_item! {
     /// Algorithm for hashing
-    #[derive(Debug, Display, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Default, DeserializeFromStr, SerializeDisplay, Decode, Encode, FfiType, IntoSchema)]
+    #[derive(Debug, Display, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Default, DeserializeFromStr, SerializeDisplay, Decode, Encode, IntoSchema)]
```

Comment:
This very likely would have been formatted with `rustfmt` if it weren't inside a function-like macro. Can we momentarily pull these out, format, and then put back in?